backend/Routes/Controllers/auth/index.js

```diff
@@ -0,0 +1,278 @@
+const auth = require('../../../Models/auth');
+const middleware = require('../auth/middlewares');
+const ajv = require('../../../Schema');
+const express = require('express');
+const router = express.Router();
+
+const {
+  signupSchema,
+  loginSchema,
+  updateUserSchema,
+  updatePasswordSchema,
+  forgotPasswordSchema,
+  resetPasswordSchema,
+} = require('../../../Schema/auth');
+
+router.post('/signup', (req, res) => {
+  let validate = ajv.compile(signupSchema);
+  let valid = validate(req.body);
+  if (!valid) {
+    return res.status(400).json({
+      success: false,
+      error: validate.errors,
```

Comment:
I guess this will return an array, you can use reduce function to convert it into a string,

Reply:
Same goes for all the other endpoints.